packages/client/src/components/DateRangePicker.tsx

```diff
@@ -42,13 +42,11 @@ import endOfMonth from 'date-fns/endOfMonth'
 
 const { useState, useEffect, useMemo } = React
 
-const LIMIT_YEAR_PAST_RECORDS = 1900
-
 function getMonthsShort(locale = 'en') {
   const months = []
   for (let i = 0; i < 12; i++) {
     months.push(
-      new Date(1970, i).toLocaleString(locale, {
+      new Date(1870, i).toLocaleString(locale, {
```

Comment:
are months in 1870 different than those in 1970? :smile:

Reply:
@bvenceslas There is a better way to form a list of month names. I'm unfamiliar with the origin story of this function but I'm fairly sure 1970 here only signifies the start of Unix time.

Reply:
> are months in 1870 different than those in 1970? 😄

I just guessed 😂